documentation/docs/UI/setup/_category_.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
What's the reason behind moving it to `setup` folder?

Reply:
I was just moving it away before the `setup` is complete. Let me revert then, there's a PR for setup` `already